circle.yml

```diff
@@ -19,5 +19,8 @@ dependencies:
     - cd wp-e2e-tests && npm install
 
 test:
+  pre:
+    - curl https://calypso.live/?branch=$BRANCHNAME
+    - until $(curl https://calypso.live/?list_branches 2>/dev/null | jq '.ready | contains( ["try/live-branch-test-pr-only"]  )'); do sleep 5; done
```

Comment:
$BRANCHNAME doesn't seem to work here (or on the command line locally) and I am not sure why.

@hoverduck: any ideas?

Reply:
The single quotes in the parameter to `jq` are preventing environment variable expansion.  Try

`curl https://calypso.live/?list_branches 2>/dev/null | jq ".ready | contains( [\"$BRANCHNAME\"]  )"`